apps/dashboard/test/config/configuration_singleton_test.rb

```diff
@@ -391,4 +391,14 @@ def config_fixtures
     cfg.stubs(:config).returns({})
     assert_equal "", cfg.pinned_apps_group_by
   end
+
+  test "api is disabled by default" do
+    ENV.delete("OOD_BATCH_CONNECT_API_ENABLED")
+    refute ConfigurationSingleton.new.batch_connect_api_enabled?
+  end
```

Comment:
We use this pattern a lot where we do stuff in blocks passed `with_modified_env`.  You can inline the environment parameter, but I would like to be consistent in using this `with_modified_env` instead of having to care/worry about the state of the ENV from test to test.

https://github.com/OSC/ondemand/blob/aa5bc6269d0aa70e4e7056793bb97aaf95979dc2/apps/dashboard/test/integration/pinned_apps_test.rb#L116-L123

Reply:
OK - will update